### PR TITLE
docs: Putaway Rule

### PR DESCRIPTION
**Pull request by marination**
Documentation for https://github.com/frappe/erpnext/pull/23969

screencapture-develop-in-8004-docs-user-manual-en-stock-putaway-rule-2021-02-08-13_42_17

ToDo:

- [x] Add Warehouse Capacity Summary Docs

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Before creating and using a Putaway Rule, it is advised that you create the following first:
+
+- [Stock Item](/docs/user/manual/en/stock/item)
+- [Warehouse](/docs/user/manual/en/stock/warehouse)
+
+## 2. How to create a Putaway Rule
+
+1. Go to the Putaway Rule list, click on New.
+ <img class='screenshot' alt='Unsaved Pick List' src='{{docs_base_url}}/assets/img/stock/unsaved-putaway-rule.png'>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:35]:
```suggestion
 <img class='screenshot' alt='Unsaved Putaway Rule' src='{{docs_base_url}}/assets/img/stock/unsaved-putaway-rule.png'>
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Set the Company and Select an Item.
+1. Set the Warehouse on which this rule is applicable.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:38]:
```suggestion
1. Select the Warehouse on which this rule is applicable.
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+1. Set the Capacity. You can also select a UOM if you want to set the Capacity in a different UOM. The Capacity in Stock UOM will be set automatically.
+ <img class='screenshot' alt='Unsaved Pick List' src='{{docs_base_url}}/assets/img/stock/multi-uom-putaway-rule.png'>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:40]:
```suggestion
 <img class='screenshot' alt='Multi UOM Putaway Rule' src='{{docs_base_url}}/assets/img/stock/multi-uom-putaway-rule.png'>
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Set the Priority. This can begin from 1 onwards, 1 being the highest priority.
+1. Save.
+ <img class='screenshot' alt='Unsaved Pick List' src='{{docs_base_url}}/assets/img/stock/saved-putaway-rule.png'>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:44]:
```suggestion
 <img class='screenshot' alt='Saved Putaway Rule' src='{{docs_base_url}}/assets/img/stock/saved-putaway-rule.png'>
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+1. You can additionally Disable a Putaway Rule as well.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:46]: how? is there a disable checkbox? Add that too in the documentation

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## 3. How does it work
+
+As mentioned above the Putaway Rules are applied on the creation of a **Purchase Receipt** from a **Purchase Order**. Let us see that in action:
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:50]:
```suggestion
As mentioned above, the Putaway Rules are applied on the creation of a **Purchase Receipt** from a **Purchase Order**. Let us see that in action:
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+1. We have a Purchase Order with a requirement of 2 Bags (2000 Kgs) of Rice:
+ <img class='screenshot' alt='Unsaved Pick List' src='{{docs_base_url}}/assets/img/stock/po-putaway-demo.png'>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:53]:
```suggestion
 <img class='screenshot' alt='Putaway Rule in Purchase Order' src='{{docs_base_url}}/assets/img/stock/po-putaway-demo.png'>
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+1. We have 2 active Putaway Rules below. One has more priority than the other:
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:55]:
```suggestion
1. We have 2 active Putaway Rules below. One has a higher priority than the other:
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+ <img class='screenshot' alt='Unsaved Pick List' src='{{docs_base_url}}/assets/img/stock/active-putaway-rules-list.png'>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:56]:
```suggestion
 <img class='screenshot' alt='Active Putaway Rules List' src='{{docs_base_url}}/assets/img/stock/active-putaway-rules-list.png'>
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+1. On creating a Purchase Receipt from this Purchase Order, one row of 2 Bags is split and assigned according to the rules. First, the Item is accommodated in 'Stores - HUL'. Once this Warehouse is at capacity, it assigns the rest to 'Finished Goods - HUL':
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:58]:
```suggestion
1. On creating a Purchase Receipt from this Purchase Order, one row of 2 Bags is split and assigned according to the rules. First, the Item is accommodated in the 'Stores - HUL' warehouse. Once this warehouse is at capacity, it assigns the rest to the 'Finished Goods - HUL' warehouse.
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md (hunk 8 above)
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:55]:
```suggestion
1. We have 2 active Putaway Rules below. One has more priority than the other.
```

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+ <img class='screenshot' alt='Unsaved Pick List' src='{{docs_base_url}}/assets/img/stock/putaway-po-to-pr.gif'>
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:59]:
```suggestion
 <img class='screenshot' alt='Putaway Rules in Purchase Order to Purchase Receipt' src='{{docs_base_url}}/assets/img/stock/putaway-po-to-pr.gif'>
```

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md (hunk 5 above)
Comment (marination) [reply erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:46]: @michellealva it's in the image above. Its visible. I don't think it requires explanation. Its just a checkbox thta we have elsewhere in the product

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md (hunk 8 above)
Comment (marination) [reply erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:55]: seems okay to me, its consistent with the previous point

File: erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+# Putaway Rule
+
+**A Putaway Rule defines a Warehouse Assignment Strategy for incoming stock.**
+
+A Putaway Rule is uniquely defined for an Item-Warehouse combination in a Company. It takes Warehouse Capacity and Priority into consideration.
+
+While creating a **Purchase Receipt** from a **Purchase Order**, the Putaway Rules are applied and Items are **auto-assigned** to Warehouses based on the given strategy.
+
+This is particularly useful for capacity management in large warehouses with multiple locations.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/stock/putaway-rule.md:19]: should add more here about temporary storing then 'putaway'

**Review by michellealva — CHANGES_REQUESTED** [review]
delete erpnext_documentation/www/docs/assets/img/stock/putaway-po-to-pr.gif

**Review by michellealva — APPROVED** [review]

**Pull request merged**